### PR TITLE
doc: default values for mon_health_to_clog_* were flipped

### PR DESCRIPTION
**Pull request by falcon78921**
## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

On a freshly installed nautilus cluster (i.e. 14.2.2), the default values are:

~~~
mon_health_to_clog_interval = 3600
mon_health_to_clog_tick_interval = 60.000000
~~~

Fixes: https://tracker.ceph.com/issues/41403
Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by falcon78921** [comment]
jenkins retest this please

File: doc/rados/configuration/mon-config-ref.rst [hunk 1]
```diff
@@ -423,7 +423,7 @@ by setting it in the ``[mon]`` section of the configuration file.
               is empty or identical to the last time, monitor will not send it
               to cluster log.
 :Type: Integer
-:Default: 3600
+:Default: 60.000000
```
Comment (tchaikov) [inline doc/rados/configuration/mon-config-ref.rst]: @falcon78921 could you take this opportunity to change the "Type" of this setting to "Float"?

Comment (falcon78921) [reply doc/rados/configuration/mon-config-ref.rst]: @tchaikov thanks! :)

Comment (tchaikov) [reply doc/rados/configuration/mon-config-ref.rst]: thank you!

**Pull request merged**